packages/v3/vite.config.ts

```diff
@@ -0,0 +1,93 @@
+import { defineConfig } from "vite";
+import gzipPlugin from "rollup-plugin-gzip";
+import minifyHTML from "rollup-plugin-minify-html-template-literals";
+import { brotliCompressSync } from "zlib";
+import { nodeResolve } from "@rollup/plugin-node-resolve";
+import loadVersion from "vite-plugin-package-version";
+import { viteSingleFile } from "vite-plugin-singlefile";
+import { minifyHtml as ViteMinifyHtml } from "vite-plugin-html";
+import stripBanner from "rollup-plugin-strip-banner";
+import replace from "@rollup/plugin-replace";
+
+const proxy_target = process.env.PROXY_TARGET || "http://nodemcu.local";
+
+export default defineConfig({
+  clearScreen: false,
+  plugins: [
+    {
+      ...nodeResolve({ exportConditions: ["development"] }),
+      enforce: "pre",
+      apply: "start",
+    },
+    stripBanner(),
+    loadVersion(),
+    { ...minifyHTML(), enforce: "pre", apply: "build" },
+    //
+    {
+      ...ViteMinifyHtml({ removeComments: true }),
+      enforce: "post",
+      apply: "build",
+    },
+    replace({
+      "@license": "license",
+      "Value passed to 'css' function must be a 'css' function result:":
+        "use css function",
+      "Use 'unsafeCSS' to pass non-literal values, but take care to ensure page security.":
+        "Use unsafeCSS",
+      delimiters: ["", ""],
+      preventAssignment: true,
+    }),
+    viteSingleFile(),
+    {
+      ...gzipPlugin({
+        filter: /\.(js|css|html|svg)$/,
+        additionalFiles: [],
+        customCompression: (content) =>
+          brotliCompressSync(Buffer.from(content)),
+        fileName: ".gz",
+      }),
+      enforce: "post",
+      apply: "build",
+    },
+    // {
+    //   ...gzipPlugin({ filter: /\.(js|css|html|svg)$/ }),
+    //   enforce: "post",
+    //   apply: "build",
+    // }
```

Comment:
@afarago Why was this removed? Seems like it is resulting in the following files not being created:
- www.js.br
- index.html.br